docs/cri/registry.md

```diff
@@ -2,6 +2,14 @@
 
 This document describes the method to configure the image registry for `containerd` for use with the `cri` plugin.
 
+*** registry.mirrors and registry.configs as described in this document
+have been DEPRECATED. As described in [the cri config](./config.md#registry-configuration) you
+should now use the form
+```toml
+[plugins."io.containerd.grpc.v1.cri".registry]
+   config_path = "/etc/containerd/certs.d"
```

Comment:
Can we mention that this is the default and there is no need to set it unless they want to change it?

Reply:
Or maybe its not default for cri...

Reply:
[plugins."io.containerd.grpc.v1.cri".registry]
      config_path = ""

Current default ^

Reply:
That's right we didn't turn it on by default because of the other registry configs.

Reply:
i'm thinking this file is just for the now deprecated containerd cri 1.4 registry config.. and the config.md can have a section for 1.5 registry configuration.. (at least for now)

Reply:
> That's right we didn't turn it on by default because of the other registry configs.

amended the last commit to be more accurate about what the default is now.. and explain more about how to change it up.